test/index.js

```diff
@@ -28,3 +34,28 @@ test('errors on too low version', async t => {
 	await t.throwsAsync(np('1.0.0', defaultOptions), /New version `1\.0\.0` should be higher than current version `\d+\.\d+\.\d+`/);
 	await t.throwsAsync(np('1.0.0-beta', defaultOptions), /New version `1\.0\.0-beta` should be higher than current version `\d+\.\d+\.\d+`/);
 });
+
+test('should not enable 2fa if the package exists', async t => {
+	const enable2faStub = sinon.stub();
+	const np = proxyquire('../source', {
+		del: sinon.stub(),
```

Comment:
Why do we need all of these stubs? Can't we run np with `--yolo` and remove the need for most of them? While I do want the tests to reflect actual stability, I think it's more important in this case to be as specific as possible, and only test what we need to verify - that we skip enabling 2FA when not needed to do so.

Reply:
`--yolo` can only remove `del` and `execa`. I used to want to use `--preview`, but it would skip the enable2fa task totally. And I don't think adding stubs will affect the correctness of the test case. Maybe because I consider it as unit test, while you treat it as end-to-end test.